kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -132,31 +139,95 @@
     },
     mixins: [commonCoreStrings],
     setup() {
+      const store = getCurrentInstance().proxy.$store;
+      const topicId = store.state.route.params.topic_id;
```

Comment:
I think you'll need to wrap this in `computed(() => store.state.route.params.topic_id)` for it to be reactive, and hence watchable later.